.test-defs/RegisterSeed.yaml

```diff
@@ -0,0 +1,30 @@
+kind: TestDefinition
+metadata:
+  name: register-seed
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the registration of a seed.
+  activeDeadlineSeconds: 7200
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/system/seed_registration
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
+    -kubecfg=$TM_KUBECONFIG_PATH/gardener.config
+    -seed-name=$SEED_NAME
+    -shoot-kubecfg=$TM_KUBECONFIG_PATH/shooted-seed.config
+    -secret-binding=$SECRET_BINDING
+    -shoot-name=$SHOOT_NAME
+    -shoot-namespace=$PROJECT_NAMESPACE
+    -backup-secret-provider=$BACKUP_SECRET_PROVIDER
+    -provider-type=$PROVIDER_TYPE
+    -provider-region=$PROVIDER_REGION
+    -networking-default-shoot-services=$NETWORKING_DEFAULT_SERVICES
+    -networking-default-shoot-pods=$NETWORKING_DEFAULT_PODS
+    -networking-blockcidrs=$NETWORKING_BLOCKCIDRS
+    -networking-pods=$NETWORKING_PODS
+    -networking-services=$NETWORKING_SERVICES
+    -networking-nodes=$NETWORKING_NODES
```

Comment:
Isn't the proper way to get `-networking-pods`, `-networking-services`, `-networking-nodes` is to lookup the Shoot spec? Is this test is Shoot agnostic and not in all cases there is a passed Shoot?

Reply:
Initially we thought about getting the networking from the shoot, but since the shoot and seed creation share a test run they will share the environment variables for the networks.

Reply:
Sure. So you vote to have them explicitly defined and passed to the test (without lookup in the Shoot spec)?

Reply:
ping @plkokanov

Reply:
Yep I'd go for having them explicitly defined